terminus-ui/table/src/table.component.html

```diff
@@ -1,8 +0,0 @@
-<div class="ts-table__wrapper">
```

Comment:
This was missed in my previous PR. I plan on rebasing this into the correct commit once this PR is merged into v15

Reply:
this block not needed? just making sure...

Reply:
Yeah we used to need a custom wrapper so we defined our own template. Now we don't need that so we're just using the CDK table template. I had already changed the component to reference the CDK template, but forgot to remove the old template file.

Reply:
Yeah we used to need a custom wrapper so we defined our own template. Now we don't need that so we're just using the CDK table template. I had already changed the component to reference the CDK template, but forgot to remove the old template file.